src/runtime/tests/manifest-parser-test.ts

```diff
@@ -466,6 +466,42 @@ describe('manifest parser', () => {
         kotlinThings: reads KotlinThings {aByte: Byte, aShort: Short, anInt: Int, aLong: Long, aChar: Char, aFloat: Float, aDouble: Double}
     `);
   });
+  it('parses a schema with a nested schema', () => {
+    parse(`
+      schema ContainsNested
+        fieldBefore: Double
+        nestedField: inline * {someNums: List<Number>, aString: Text}
+        fieldAfter: List<Long>
+    `);
+  });
+  it('parses an inline schema with a nested schema', () => {
```

Comment:
I think these test names need to be updated. This isn't an inline schema with a nested schema; it's a schema with an inline nested schema.